File: app/src/main/java/com/anokmik/databinding/fragment/FindViewByIdFragment.java
package com.anokmik.databinding.fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.anokmik.databinding.R;
import com.anokmik.databinding.model.User;

public class FindViewByIdFragment extends BaseFragment {

    private TextView firstName;

    private TextView lastName;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_simple, container, false);
        firstName = (TextView) view.findViewById(R.id.first_name);
        lastName = (TextView) view.findViewById(R.id.last_name);
        return view;
    }

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        User user = User.getDefault();
        firstName.setText(user.firstName);
        lastName.setText(user.lastName);
    }

}

File: app/src/main/kotlin/com/anokmik/databinding/fragment/FindViewByIdFragment.kt
package com.anokmik.databinding.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView

import com.anokmik.databinding.R
import com.anokmik.databinding.model.User

class FindViewByIdFragment : BaseFragment() {

    private val firstName by lazy { view?.findViewById(R.id.first_name) as TextView }

    private val lastName by lazy { view?.findViewById(R.id.last_name) as TextView }

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater?.inflate(R.layout.fragment_simple, container, false)
    }

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        val user = User.DEFAULT
        firstName.text = user.firstName
        lastName.text = user.lastName
    }

}